src/jl_uv.c

```diff
@@ -754,6 +754,7 @@ STATIC_INLINE void write_to_safe_crash_log(char *buf) JL_NOTSAFEPOINT
 }
 
 extern int jl_inside_heartbeat_thread(void);
+extern uv_thread_t safepoint_waiter;
```

Comment:
I think using an interface function is better: `jl_inside_safepoint_waiter`?

Reply:
Done!